### ISSUE TITLE
[Plus] Opt in switch for feature #2531 to prevent data loss

### ISSUE THREAD

**Issue by RandomGOTI** [sandboxie-plus/Sandboxie#2553]
### Is your feature request related to a problem or use case?

The feature at #2531 can result in data loss if there is no switch to control whenever the users desires it or not
There are use cases allowed by the AutoDelete to postpone/cancel it having the option to keep the files as is untill the next time it is triggered and as the AutoDelete only triggers only after a box session ends the new feature should be opt in as it can easily cause data loss for users that are experienced with Sandboxie but unaware of the added feature.

Even configuring a not empty box to AutoDelete but not yet deleting the content or recover it would result in losing it , since how AutoDelete triggers (not starting box session posses no risk for its contents)

### Describe the solution you'd like

>title

### Describe alternatives you've considered

switch to "manual" box cleaning

**Comment by DavidXanatos** [comment]
There is a bug to be mindful of: https://github.com/sandboxie-plus/Sandboxie/issues/2555

Other than that the implementation of the feature should avoid issues, when triggered it behaves as if the last program would have just closed so if the user was offered a dialog to keep files that dialog will be shown again.

The scenario where a user sets a not empty box to be auto deleted and then restarts sandman is not covered, but the assumption is that then the user wants the box content to be gone. 

I'll add an option to disable toe box clean up on start up into the next build.
